src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+import java.util.Random;
+
 public class Duke {
+
+    public static void printSeparator() {
+        for(int i = 0; i < 40; i++) {
+            System.out.print('-');
+        }
+        System.out.print('\n');
+    }
+
+    public static void exitMethod() {
+        System.out.print("My cover's blown!\n");
+        printSeparator();
+    }
+
+    public static void echoMode(String line) {
+        Random rd = new Random();
```

Comment:
Using "rd" as the Random object name seems to be vague and might not go by the naming convention. Instead you can use a more specific word like randomNumber that provides better readability and explanation to what you are intending to do.

Reply:
rd is not a random number, it can be used to generate random integers or floats too, depending on the method that is called. in this case, I called a .nextBoolean method. Not really sure what to name this random object